tests/e2e/e2e_test.go

```diff
@@ -22,9 +22,9 @@ import (
 )
 
 const (
-	egLatest      = "v0.0.0-latest" // This defaults to the latest dev version.
-	egNamespace   = "envoy-gateway-system"
-	egDefaultPort = 10080
+	egDefaultVersion = "v1.4.0"
```

Comment:
i switched the default local version to the stable one as it can be a source of flake (if something broken on EG repo)